fees/betswirl/index.ts

```diff
@@ -0,0 +1,283 @@
+import { request } from "graphql-request";
+import BigNumber from "bignumber.js";
+
+import { api } from "@defillama/sdk";
+import { BSC, POLYGON, AVAX } from "../../helpers/chains";
+import { getTimestampAtStartOfDayUTC } from "../../utils/date";
+import { Chain } from "@defillama/sdk/build/general";
+import { getPrices } from "../../utils/prices";
+
+const ten = toBN("10");
+function fromWei(wei: string | BigNumber, unit = 18) {
+  return toBN(wei).dividedBy(ten.pow(unit));
+}
+function toBN(wei: string | BigNumber) {
+  return new BigNumber(wei);
+}
+
+const endpoints: any = {
+  [BSC]: "https://api.thegraph.com/subgraphs/name/betswirl/betswirl-bnb",
+  [POLYGON]:
+    "https://api.thegraph.com/subgraphs/name/betswirl/betswirl-polygon",
+  [AVAX]: "https://api.thegraph.com/subgraphs/name/betswirl/betswirl-avalanche",
+};
+
+function graphs() {
+  return (chain: Chain) => {
+    return async (timestamp: number) => {
```

Comment:
i think this func still returns wrong type, can you check or add `  return async (timestamp: number): Promise<FetchResultFees> => {` for check the expected type